src/index.css

```diff
@@ -3,3 +3,7 @@
   max-width: min-content;
   margin: auto;
 }
+
+img {
+  max-width: 90vw;
+}
```

Comment:
Perhaps doesn't make as much of an impact as you might think, because the image is strictly limited in size by the div around it at the moment. You might like playing around with using % as a unit around your css files which is a powerful way to create layouts that are responsive and adapt to different screen sizes, because it makes the size depend on the parent element